src/actions/graphql.js

```diff
@@ -1,6 +1,8 @@
+import { mapValues, get, flow } from 'lodash/fp'
 import { SEND_GRAPHQL_QUERY } from './index'
 
-export function sendGraphqlQuery (subject, id, query, variables) {
+export function sendGraphqlQuery (subject, id, query, opts) {
+  const { variables, addDataToStore } = opts
```

Comment:
please change my use of `sendGraphqlQuery` in People.js to work with this change